### PR TITLE
feat: enable passing ring width from mobile

### PR DESCRIPTION
**Pull request by yqrashawn**
-------

mobile issue https://github.com/status-im/status-mobile/issues/15037
mobile pr https://github.com/status-im/status-mobile/pull/17034

ident ring width can be 2 or 4 in design system
this PR enables reading ring width from mobile via image server handler

### CODE REVIEW

File: server/handlers.go [hunk 1]
```diff
@@ -278,6 +289,12 @@ func handleAccountImagesImpl(multiaccountsDB *multiaccounts.Database, logger *za
 		logger.Error("handleAccountImagesImpl: failed to load image.", zap.String("keyUid", parsed.KeyUID), zap.String("imageName", parsed.ImageName), zap.Error(err))
 		return
 	}
+
+	if parsed.Ring && parsed.RingWidth == 0 {
+		logger.Error("handleAccountImagesImpl: no ringWidth.")
+		return
```
Comment (qfrank) [inline server/handlers.go:293]: this return makes me feel following fallback useless. if `RingWidth` is required, perhaps we can validate it within function `DrawRing` and return the error.
```
if param.RingWidth == 0 {
    ringPxSize = math.Max(2.0, float64(param.Width/16.0))
}
```

**Review by qfrank — APPROVED** [review]
LGTM, left minor comment ❤️

File: server/handlers.go (hunk 1 above)
Comment (yqrashawn) [reply server/handlers.go:293]: this is for other usage of `DrawRing`, like this one https://github.com/status-im/status-go/blob/f7530f3cd0224ae9b13728741938980446505337/server/handlers.go#L724
although I don't know if this handler is being used anywhere

Comment (qfrank) [reply server/handlers.go:293]: proberbly we don't use it anymore , we can remove relate code to `status-go` safely, we can check the git history for file `src/utils/image_server.cljs` , relate commit id: `ce322c9537dff1905d60ce70adc9fd378cf885eb` , relate [PR](https://github.com/status-im/status-mobile/pull/15815)

Comment (yqrashawn) [reply server/handlers.go:293]: you are right, i checked mobile and desktop code. `handleIdenticon` is not used. I added a commit to remove it.

**Review by alwx — APPROVED** [review]

**Pull request merged**